core/examples/http_bench_json_ops.rs

```diff
@@ -196,5 +195,5 @@ fn main() {
       .unwrap();
     isolate.await
   };
-  js_check(runtime.block_on(future));
+  runtime.block_on(future).unwrap();
```

Comment:
So I put a syntax error into `core/examples/http_bench_json_ops.js` to see what the error message would look like:
```
> ./target/debug/examples/http_bench_json_ops
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: SyntaxError: Unexpected string', core/examples/http_bench_json_ops.rs:195:8
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```
It gives me something vaguely reasonable, but not a line number.

Reply:
runtime error (throwing in listen function) gave a good result:
```
> ./target/debug/examples/http_bench_json_ops
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Error: foo
    at listen (http_bench_json_ops.js:14:9)
    at main (http_bench_json_ops.js:62:23)
    at http_bench_json_ops.js:75:1', core/examples/http_bench_json_ops.rs:198:28
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

Reply:
The location for syntax errors can only be found in the "source line" info. This might help: https://github.com/denoland/deno/blob/f601721851bdd64ee5b40bd3d408df063691b425/cli/fmt_errors.rs#L259-L273 This is a pretty way of handling it, to present the source line info as if it were a single stack frame when there are no actual stack frames. I guess just duplicate this in `core::error::JsError::fmt`.

(For runtime errors, the source line info usually just aligns with the top stack frame and is redundant.)

Reply:
Nice catch. Thanks for advice Nayeem, updating...

Reply:
Done, syntax error for example:
```
Uncaught SyntaxError: Invalid or unexpected token
    at error_without_stack.js:3:14
```